src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -159,46 +165,47 @@ we recommend specifying a version range using one of the following formats:
     collection: '^0.1.2'
   ```
 
-For additional details, see the [Pub versioning
+For additional details, see the [package versioning
 guide]({{site.dart-site}}/tools/pub/versioning).
 
 ### Updating package dependencies
 
-When you run `flutter pub get` ('Packages Get' in IntelliJ) for
-the first time after adding a package, Flutter saves the concrete package
-version found in the `pubspec.lock`
+When running `flutter pub get` (**Packages Get** in IntelliJ)
+for the first time after adding a package,
+Flutter saves the concrete package version found in the `pubspec.lock`
 [lockfile]({{site.dart-site}}/tools/pub/glossary#lockfile).
 This ensures that you get the same version again if you, or another
 developer on your team, run `flutter pub get`.
 
-If you want to upgrade to a new version of the package,
+To upgrade to a new version of the package,
 for example to use new features in that package, run
```

Comment:
For ease of reading, consider using em-dashes or parentheses instead of commas to set off the "for example...".